.github/workflows/build-pr.yml

```diff
@@ -203,4 +203,4 @@ jobs:
 
       - name: clean up
         run: |
-          make e2e-cleanup AZURE_SUBSCRIPTION_ID=${{ env.AZURE_SUBSCRIPTION_ID }}
```

Comment:
if there is no actual change to build-pr.yml and Makefile, could you just check out them from main branch?